unity-renderer/Assets/Scripts/MainScripts/DCL/Bridges/RealmsInfoBridge/RealmsInfoBridge.cs

```diff
@@ -25,5 +25,10 @@ public void ConnectionToRealmFailed(string json)
             var realmConnectionFailedPayload = Utils.SafeFromJson<JumpInPayload>(json);
             OnRealmConnectionFailed?.Invoke(realmConnectionFailedPayload);
         }
+        
+        public void SetRealmAbout(string payload)
```

Comment:
you can declare `[PublicAPI]`

Reply:
Done!

Reply:
I dont know why, this does not reflect the latest code. There is a github bug going on here. But I have already added the [PublicAPI] annotation.